### ISSUE TITLE
Bug Report:  pip install psychopy not working on anaconda

### ISSUE THREAD

**Issue by yaowang1111** [psychopy/psychopy#5016]
I'm using the anaconda 2022.05 version and tring to use pip to install psychopy 2022.2.2,  and the following error pompt out. My python version is 3.8.13.  I have updated pip and setuptools, not working.

```
Collecting psychopy
  Using cached PsychoPy-2022.2.2.zip (26.2 MB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [16 lines of output]
      C:\Users\yaowang\AppData\Local\Temp\pip-install-fk1u3y2l\psychopy_1212150a9b12403da57ad41d3c4c67b4\setup.py:28: SetuptoolsDeprecationWarning: As setuptools moves its configuration towards `pyproject.toml`,
      `setuptools.config.read_configuration` became deprecated.

      For the time being, you can use the `setuptools.config.setupcfg` module
      to access a backward compatible API, but this module is provisional
      and might be removed in the future.

        required = read_configuration('setup.cfg')['options']['install_requires']
      C:\Users\yaowang\anaconda3\envs\psychopy\lib\site-packages\setuptools\config\setupcfg.py:463: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
        warnings.warn(msg, warning_class)
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\yaowang\AppData\Local\Temp\pip-install-fk1u3y2l\psychopy_1212150a9b12403da57ad41d3c4c67b4\setup.py", line 41, in <module>
          required.remove('pyqt5; python_version >= "3"')
      ValueError: list.remove(x): x not in list
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by yaowang1111** [comment]
I reinstalled my computer, reinstalled the newest version of anaconda, tried 3.10 version of python, nothing helps, the error is still there. So i guess there is some bug with psychopy, please help me solve it.

**Comment by trentwirth** [comment]
I had the same issue

**Comment by snastase** [comment]
I'm also encountering this error when trying to `pip install psychopy` in a dedicated (mini)conda environment with `Python 3.8.13`, `conda 4.12.0`, `pip 22.1.2`:

```
pip install psychopy --no-cache-dir
Collecting psychopy
  Downloading PsychoPy-2022.2.2.zip (26.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 26.2/26.2 MB 86.9 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      /Users/snastase/opt/miniconda3/envs/psychopy/lib/python3.8/site-packages/setuptools/config/__init__.py:28: SetuptoolsDeprecationWarning: As setuptools moves its configuration towards `pyproject.toml`,
      `setuptools.config.read_configuration` became deprecated.
      
      For the time being, you can use the `setuptools.config.setupcfg` module
      to access a backward compatible API, but this module is provisional
      and might be removed in the future.
      
        warnings.warn(dedent(msg), SetuptoolsDeprecationWarning)
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/ch/f_vp72713hg84pcp6t_2wfmw0000gp/T/pip-install-1t131k3h/psychopy_b16dc10c0f1a4feda700676513723794/setup.py", line 41, in <module>
          required.remove('pyqt5; python_version >= "3"')
      ValueError: list.remove(x): x not in list
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

I tried a manual `pip install pyqt5`, but having `pyqt5` installed doesn't seem to make a difference.

**Comment by oreHGA** [comment]
+1 on this issue, trying to use older versions to see if they work for my package but no luck yet

**Comment by khl02007** [comment]
Hi guys, installing directly from the repo (as opposed to from PyPI) worked for me (git clone this repo and do `pip install -e path/to/repo`) in case anyone is still looking for a workaround. But the devs should probably release a fix on PyPI.

**Comment by glupyan** [comment]
Had same issue with 2022.2.2
pip install psychopy==2022.1.4 works

**Comment by peircej** [comment]
I believe this was fixed by https://github.com/psychopy/psychopy/pull/5017 14 days ago, which is why @khl02007 found that it works fine using the repo. This fix has been included in the 2022.2.3 release, which is now on pypi, so this should be working from pypi now, but re-open the issue if it still fails
